src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -316,6 +316,24 @@ public boolean isClosingWithinTimeLimit(int hours) {
                && differenceBetweenDeadlineAndNow >= hours - 1
                && differenceBetweenDeadlineAndNow < hours;
     }
+    
+    /**
+     * returns true if the session is closed within the past hour of calling this function
```

Comment:
Is there a possibility that a session might return true to two consecutive calls of this method because this method is called exactly the same time as the deadline?